tutorials/02_install.md

```diff
@@ -30,6 +20,27 @@ Install Ignition Rendering:
     # This installs ign-rendering3. Change the number after libignition-rendering to the version you want
     sudo apt install libignition-rendering3-dev
 
+### Windows
+
+Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+Miniconda suffices.
+
+Create if necessary, and activate a Conda environment:
+
+```
+conda create -n ign-ws
+conda activate ign-ws
+```
+
+Install:
+
+```
+conda install libignition-rendering<#> --channel conda-forge
```

Comment:
`libignition-rendering` isn't currently in `conda-forge`, should we hold off on adding the Windows binary installation for now?